gcp/iap-ingress/base/config-map.yaml

```diff
@@ -1,349 +1,74 @@
-                                                    "clusters": [
-                                                        {
-                                                            "name": "cluster_istiogateway",
-                                                            "weight": 100
-                                                        }
-                                                    ]
-                                                }
-                                            },
-                                            {
-                                                "prefix": "/",
-                                                "prefix_rewrite": "/",
-                                                "timeout_ms": 10000,
-                                                "use_websocket": true,
-                                                "weighted_clusters": {
-                                                    "clusters": [
-                                                        {
-                                                            "name": "cluster_ambassador",
-                                                            "weight": 100
-                                                        }
-                                                    ]
-                                                }
-                                            }
-                                        ]
-                                    }
-                                ]
-                            },
-                            "stat_prefix": "ingress_http"
-                        },
-                        "name": "http_connection_manager",
-                        "type": "read"
-                    }
-                ]
-            }
-        ],
-        "stats_flush_interval_ms": 1000,
-        "statsd_udp_ip_address": "127.0.0.1:8025"
-    }
+  healthcheck_route.yaml: |
+    apiVersion: networking.istio.io/v1alpha3
+    kind: VirtualService
+    metadata:
+      name: default-routes
+      namespace: $(namespace)
+    spec:
+      hosts:
+      - "*"
+      gateways:
+      - kubeflow-gateway
+      http:
+      - match:
+        - uri:
+            exact: /healthz
+        route:
+        - destination:
+            port:
+              number: 80
+            host: whoami-app.kubeflow.svc.cluster.local
+      - match:
+        - uri:
+            exact: /whoami
+        route:
+        - destination:
+            port:
+              number: 80
+            host: whoami-app.kubeflow.svc.cluster.local
+    ---
+    apiVersion: networking.istio.io/v1alpha3
+    kind: Gateway
+    metadata:
+      name: kubeflow-gateway
+      namespace: $(namespace)
+    spec:
+      selector:
+        istio: ingressgateway
+      servers:
+      - port:
+          number: 80
+          name: http
+          protocol: HTTP
+        hosts:
+        - "*"
+  jwt-policy-template.yaml: |
```

Comment:
this is recently changed:
- added to ingress/base: https://github.com/kubeflow/manifests/pull/69
- removed from setup-backend and patch audience info later: https://github.com/kubeflow/kubeflow/pull/3298